server/repository/migrations/sqlite/2021-09-17T10-00_create_requisition_table/up.sql

```diff
@@ -18,5 +18,8 @@ CREATE TABLE requisition (
     their_reference TEXT,
     max_months_of_stock  DOUBLE PRECISION NOT NULL,
     min_months_of_stock DOUBLE PRECISION NOT NULL,
-    linked_requisition_id TEXT
+    linked_requisition_id TEXT,
+    authorisation_status TEXT,
+    program_id TEXT,
+    is_sync_update BOOLEAN NOT NULL DEFAULT FALSE
```

Comment:
true, i've made an [issue](https://github.com/openmsupply/open-msupply/issues/1519), maybe it should be something like `last_updated_by_sync` ?

Reply:
hm. maybe its ok as it is? that's the intention isn't it - 'has been updated by sync'
I was unsure when I read it first, which direction things were going in. Is this an update for sync? ( an outgoing thing - something that should be on the sync queue ) or something updated by sync? ( the thing has already happened and was done by sync )

I can't think of anything which sums it up nicely which usually means that what we have is the best it will get..
maybe `was_updated_by_sync` `updated_by_sync` `is_from_sync` 🤷